packages/react/src/patterns/blocks/LeadSpaceBlock/__stories__/LeadSpaceBlock.stories.js

```diff
@@ -0,0 +1,112 @@
+import { object, select, withKnobs, text } from '@storybook/addon-knobs';
+import LeadSpaceBlock from '../LeadSpaceBlock.js';
+import React from 'react';
+import readme from '../README.md';
+import { storiesOf } from '@storybook/react';
+
+storiesOf('Patterns (Blocks)|LeadSpaceBlock', module)
+  .addDecorator(withKnobs)
+  .addParameters({
+    readme: {
+      sidebar: readme,
+    },
+  })
+  .add('Default', () => {
+    const title = 'Continuous delivery';
+
+    const heading = text(
+      'Heading (required)',
+      'Innovate like a startup and scale for the enterprise '
+    );
+
+    const copy = `Automate your software release process with continuous delivery (CD)—the most 
+    critical part of adopting DevOps. Build, test, and deploy code changes quickly, 
+    ensuring software is always ready for deployment.`;
+
+    const mediaType = select(
+      'mediaType (optional)',
+      ['image', 'video', 'none'],
+      'image'
+    );
+
+    const video = {
+      videoId: '0_uka1msg4',
+      showCaption: true,
+    };
+
+    const image = {
+      heading: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit.',
+      image: {
+        sources: [
+          {
+            src: 'https://dummyimage.com/320x180/ee5396/161616&text=16:9',
+            breakpoint: 320,
+          },
+          {
+            src: 'https://dummyimage.com/400x225/ee5396/161616&text=16:9',
+            breakpoint: 400,
+          },
+          {
+            src: 'https://dummyimage.com/672x378/ee5396/161616&text=16:9',
+            breakpoint: 672,
+          },
+        ],
+        alt: 'Image alt text',
+        defaultSrc: 'https://dummyimage.com/672x378/ee5396/161616&text=16:9',
+      },
+    };
+
+    const mediaData = mediaType === 'image' ? image : video;
+
+    const linkListProps = {
+      heading: text('link list heading:', 'Featured products'),
+      items: object('link list items array', [
+        {
+          type: 'local',
+          copy: 'IBM Cloud Continuous Delivery',
+          cta: {
+            href: 'https://ibm.com',
+          },
+        },
+        {
+          type: 'local',
+          copy: 'UrbanCode',
+          cta: {
+            href: 'https://ibm.com',
+          },
+        },
+        {
+          type: 'download',
+          copy: 'View all products',
+          cta: {
+            href: 'https://ibm.com',
+          },
+        },
+      ]),
+    };
+
+    const types = ['local', 'external'];
+    const ctaProps = {
+      style: 'button',
+      type: 'local',
+      buttons: [
+        {
+          type: select('ContentBlock | CTA type', types, types[0]),
+          copy: text('ContentBlock | CTA copy', 'Contact sales'),
+          href: 'https://example.com/',
+        },
+      ],
+    };
+
+    return (
```

Comment:
what happened to the grid classes here?

Reply:
@annawen1  As we have the horizontal rule for 12 col and content 8 col applied with in the component, If I apply the same before calling the component its looking like compressed like this. so I left it in the original way.
Do we still need the grid?

![image](https://user-images.githubusercontent.com/21186867/82689835-1c438280-9c29-11ea-9172-090f858b7d27.png)